partiql-ast/src/main/java/org/partiql/ast/sql/SqlDialect.kt

```diff
@@ -394,19 +396,20 @@ public abstract class SqlDialect : AstVisitor<SqlBlock, SqlBlock>() {
     override fun visitExprLike(node: ExprLike, tail: SqlBlock): SqlBlock {
         var t = tail
         t = visitExprWrapped(node.value, t)
-        t = t concat if (node.not) " NOT LIKE " else " LIKE "
+        t = t concat if (node.isNot) " NOT LIKE " else " LIKE "
```

Comment:
(self-review): Lombok's default getter for `boolean` (not the boxed `Boolean`; boxed will still use `get`) will use the `is` prefix rather than `get`. Stylistically, `is` is
- more consistent with what we do in the plan already (e.g. RelUnion's `isAll`)
- default for IntelliJ when creating new getters from private fields
- if we care about conventions, JavaBeans prescribes using `is` over `get`

This change applies to other places we had used a boolean getter (e.g. identifier's `delimited`).